commands/management/add_mysql.go

```diff
@@ -116,6 +117,17 @@ type addMySQLCommand struct {
 	CreateUser             bool
 }
 
+func (cmd *addMySQLCommand) ApplyDefaults(cfg *ini.File) {
+	usernameOverride := cfg.Section("").Key("username").String()
```

Comment:
Is the section always empty?
Is this intended to work with a default `~/.my.cnf` file?
```
# cat ~/.my.cnf
[client]
host=127.0.0.1
port=12345
user=msandbox
password=msandbox
```

Reply:
good questions! I think it is better to follow defaults.
thanks for catching that!